rpm/droidmedia-devel.spec

```diff
@@ -0,0 +1,32 @@
+Name:          droidmedia-devel
+Summary:       Android media wrapper library development package
+Version:       0.20170214.0
+Release:       1
+Group:         System/Libraries
+License:       ASL 2.0
+Source0:       %{name}-%{version}.tgz
+BuildRequires: meson
+BuildRequires: ninja
```

Comment:
Have you built this with OBS, I'm just wondering if you'll need a dependency for libdl?

Reply:
libdl is needed for linking, I put it in pkg-config file. It does builds on OBS. Isn't libdl part of glibc-devel or I should BuildRequire it somehow?

Reply:
If it builds on OBS it should be fine, I just realized I hadn't tried it myself and sometimes things that build in the SDK because the dependency was already available don't on OBS.